.github/workflows/testing.yml

```diff
@@ -55,8 +55,5 @@ jobs:
           
       - name: Test with pytest
         run: |
-          pytest tests/test_preprocessing.py
-          pytest tests/test_svd.py
-          pytest tests/test_map_to_map.py
-          pytest tests/test_distribution_to_distribution.py
+          pytest tests/
```

Comment:
also, some of these tests have to happen in a certain order, so they should be explicitly stated

Reply:
yeah, I reverted that change